cygnus-ngsi/src/main/java/com/telefonica/iot/cygnus/sinks/NGSIMongoSink.java

```diff
@@ -235,11 +247,51 @@ public void aggregate(NGSIEvent cygnusEvent) {
                 
                 LOGGER.debug("[" + getName() + "] Processing context attribute (name=" + attrName + ", type="
                         + attrType + ")");
-                Document doc = createDoc(recvTimeTs, entityId, entityType, attrName, attrType, attrValue);
+                
+                Document doc;
+				if(attrMetadataStore.equals("true")){
```

Comment:
Ident have improved *a lot* :) but seem still to be different from existing one... checking style would be a good idea before merging the PR.